spec/models/issue_spec.rb

```diff
@@ -0,0 +1,28 @@
+# == Schema Information
+#
+# Table name: issues
+#
+#  id                   :integer          not null, primary key
+#  user_id              :integer          not null
+#  unit_id              :integer
+#  title                :string           not null
+#  description          :text             not null
+#  status               :integer          default(0), not null
+#  scope                :integer          default(0), not null
+#  estimated_completion :date
+#  created_at           :datetime         not null
+#  updated_at           :datetime         not null
+#  comment_count        :integer          default(0)
+#  event_count          :integer          default(0)
+#
+# Indexes
+#
+#  index_issues_on_unit_id  (unit_id)
+#  index_issues_on_user_id  (user_id)
+#
+
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.